js/app.js

```diff
@@ -70,12 +63,122 @@ function getDate() {
 }
 
 function Controls() {
-    var LocationsDropdown = {
+    let showModal = false;
+    let canteens = [];
+
+    function openStreetMap() {
+        const greenIcon = new L.Icon({
+            iconUrl: 'https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-2x-green.png',
```

Comment:
Can't we provide a local copy of this marker?
What is it licensed under?

Reply:
The GitHub repository is licensed under BSD (2-clause) https://github.com/pointhi/leaflet-color-markers/blob/master/LICENSE

So a local copy is possible. However, I'm not sure, on how to generally include the copyright notice?

Reply:
I have now changed it, and use CSS instead of the marker from this external repository